app/src/main/java/org/simple/clinic/patient/sync/PatientPayload.kt

```diff
@@ -67,7 +67,10 @@ data class PatientPayload(
     val reminderConsent: ReminderConsent,
 
     @Json(name = "deleted_reason")
-    val deletedReason: DeletedReason?
+    val deletedReason: DeletedReason?,
+
+    @Json(name = "registration_facility_id")
```

Comment:
Have we confirmed the contract for the API with the server team? If this goes live and the contract does not match, it will break sync for those app installations.

Reply:
Yes, they gave this link attached below. Looks good to me. Can you check once as well?
`https://simple-review-pr-1132.herokuapp.com/api-docs#tag/patient/paths/~1patients~1sync/post`

Reply:
I looked at the link, could not see the properties in `v4` APIs. Can you see them? 🤔

Reply:
Never mind, I saw the message on Slack about the change being in V3.